packages/p2p-v2/src/pages/my-profile/screens/MyProfileAdDetails/MyProfileAdDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import clsx from 'clsx';
+import { useAdvertiserInfo, useAdvertiserUpdate } from '@deriv/api';
+import { MyProfileAdDetailsTextarea } from './MyProfileAdDetailsTextarea';
+import './MyProfileAdDetails.scss';
+
+const MyProfileAdDetails = () => {
```

Comment:
https://github.com/binary-com/deriv-app/assets/103016120/f96805b6-b713-4fec-abab-7ec028d9895b

Reply:
after saving the button should look like the screenshot below and then disabled once it has saved tho 👀 if we dont need to implement the one below for now, we should at least disable the button 

<img width="128" alt="Screenshot 2024-01-10 at 1 39 24 PM" src="https://github.com/binary-com/deriv-app/assets/103412909/ab97e3cd-4c4c-448e-a23a-3235bd846421">

Reply:
Yeaa I didn't disable it cuz I thought we might use it in our sprint demo later to show we can update the ad details 😅

Reply:
Dood, wrong acount

Reply:
![image](https://github.com/binary-com/deriv-app/assets/25546711/f67210cb-010f-4f5b-99c9-78489cba7a15)

Reply:
alright sure sure, can you leave a comment in the component then so we can add this in future 👀